spec/cosign-verify-attestation_spec.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
why it's in spec folder? could you move it to hack or test folder and put into README. When I see spec then I expect rpm spec.